spec/features/publish/viewing_a_course_spec.rb

```diff
@@ -342,6 +340,12 @@ def then_i_should_see_the_description_of_the_salary_course
     )
   end
 
+  def and_i_should_see_the_course_as(status_tag)
+    expect(provider_courses_show_page.content_status).to have_content(
+                                                           status_tag,
```

Comment:
Could we just keep this all on one line. It looks funky broken up like this

Reply:
oops. Laziness 😅 

Done